helm/odigos/templates/ui/clusterrole.yaml

```diff
@@ -12,6 +12,18 @@ rules:
       - list
       - watch
       - patch
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - patch
+      - create
+      - delete
+      - update
```

Comment:
same here, I think this should be a role for the namespace only instead of clusterrole